.github/workflows/windows-vcpkg.yml

```diff
@@ -41,6 +41,24 @@ jobs:
       run: |
           choco install wget unzip zip --no-progress
 
+    # Downloads ccache, and copies it to "cl.exe" in order to trick cmake into using it,
+    # see ccache wiki for background on using it with MSVC:
+    # https://github.com/ccache/ccache/wiki/MS-Visual-Studio
+    - name: Install ccache
```

Comment:
Can't we use vcpkg to install ccache?

Reply:
I don't find it in vcpkg repo.
A possibility was to install it with chocolatey, like we do for wget/zip above, but it just makes it more difficult to implement the "masquerading" technique because choco will install it in it own nested folders, and we must take care of ot using the shim executable that it creates.
Installs from choco also seem slower compared to the plain download from github relase.